joomla/joomla-cms
# [4.3] Batch Copy Contacts

Pull Request for Issue #40263 .

### Summary of Changes
ensure that $published is an integer


Pull Request for Issue #40261 .

### Summary of Changes
ensure that $published is an integer


### Testing Instructions
Create a contact and then try a batch action. It should look like this
![image](https://user-images.githubusercontent.com/1296369/229098274-b85f51e6-d5fe-42ba-84c1-e06917d9c75a.png)

Then apply any filter and/or sort by any column and try to batch action again

### Actual result BEFORE applying this Pull Request
![image](https://user-images.githubusercontent.com/1296369/229098128-6ab57a70-cc2c-49aa-b6d9-57741a10e688.png)

### Expected result AFTER applying this Pull Request
![image](https://user-images.githubusercontent.com/1296369/229098299-c068e8dc-e21e-404e-bec0-804b25740cfe.png)



### Link to documentations
Please select:
